getting_started/srun.md

```diff
@@ -0,0 +1,52 @@
+## `srun`: Submitting Jobs to the Slurm Workload Manager
+
+Slurm is a job scheduling system commly used in High-Performance Computing (HPC) environments to efficiently allocate and manage computing resources among users.
+`srun` is a command-line tool used for submitting jobs to the Slurm workload manager, offering a wide range of options for customizing job submission.
```

Comment:
I would add that it essentially lets you run parallel jobs on a cluster managed by slurm (as mentioned in the [SLURM docs](https://slurm.schedmd.com/srun.html))